src/test/java/teammates/test/cases/common/TimeHelperTest.java

```diff
@@ -86,6 +88,99 @@ public void testCombineDateTime() throws ParseException{
     }
     
     @Test
+    public void testIsTimeWithinPeriod() {
+        Calendar startCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        Calendar endCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        Calendar timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        
+        // Set start time to 5 days before today and end time to 5 days after today
+        startCalendar.add(Calendar.DAY_OF_MONTH, -5);
+        endCalendar.add(Calendar.DAY_OF_MONTH, 5);
+        
+        Date startTime = startCalendar.getTime();
+        Date endTime = endCalendar.getTime();
+        Date time;
+        
+        
+        
```

Comment:
Let's not add too many line breaks. One should be enough, or at most two, to separate logical sections.